Weblate: Update translations

photoprismci opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/photoprism/photoprism?pullRequest=2099) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/photoprism/photoprism?pullRequest=2099) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/photoprism/photoprism?pullRequest=2099) it.</sub>

weblate committed:
Translated using Weblate (Danish)

Currently translated at 95.8% (504 of 526 strings)

Translation: PhotoPrism/Frontend
Translate-URL: https://translate.photoprism.app/projects/photoprism/frontend/da/

weblate committed:
Translated using Weblate (French)

Currently translated at 100.0% (526 of 526 strings)

Translation: PhotoPrism/Frontend
Translate-URL: https://translate.photoprism.app/projects/photoprism/frontend/fr/

weblate committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 99.6% (524 of 526 strings)

Translation: PhotoPrism/Frontend
Translate-URL: https://translate.photoprism.app/projects/photoprism/frontend/zh_Hans/
